Cubical/Data/Int/Properties.agda

```diff
@@ -134,9 +134,21 @@ m + negsuc n = m +negsuc n
 - negsuc n = pos (suc n)
 
 _-_ : Int → Int → Int
-m - pos zero    = m
-m - pos (suc n) = m + negsuc n
-m - negsuc n    = m + pos (suc n)
+m - n = m + (- n)
+
+-pos : ∀ n → - (pos n) ≡ neg n
+-pos zero = refl
+-pos (suc n) = refl
+
+-neg : ∀ n → - (neg n) ≡ pos n
+-neg zero = refl
+-neg (suc n) = refl
+
+double- : ∀ z → - (- z) ≡ z
```

Comment:
Maybe this should be `-Involutive`?

Reply:
Makes sense, and would be more consistent down the line. Done.